pages/data/index.vue

```diff
@@ -541,6 +564,20 @@ export default {
       this.$router.replace({ query }).then(() => {
         this.fetchResults()
       })
+    },
+
+    onMapClick: function(label) {
```

Comment:
Add JSDoc comments

Reply:
IMHO the type declaration serves the same purpose, but does a better job...
```
interface Methods {
    searchScaffoldLabel: (payload: { label: string }) => void
    closeDialog: () => void
}
```

Reply:
Very true, and I didn't think of that.